content/apps/how-to-retrieve-pim-structure.md

```diff
@@ -96,6 +101,24 @@ $response = $client->get(
 $channel = json_decode($response->getBody()->getContents(), true);
 ```
 
+```javascript [activate:NodeJS]
+
+import fetch from 'node-fetch';
```

Comment:
```suggestion
```

Advantage of the proposition: you won't have to write this line anymore for this guided tutorials and the other